src/tokens/TokenService.ts

```diff
@@ -58,7 +58,10 @@ export class TokenService {
         const token = this.getTokenBySymbol(symbol);
         const contract = new Contract(token.address, ERC20.abi, this.wallet);
 
-        const tx: ContractTransaction = await contract.approve(spender, ethers.constants.MaxUint256, { nonce });
+        const tx: ContractTransaction = await contract.approve(spender, 
+            ethers.constants.MaxUint256,
+            { nonce, gasLimit: 90000 }
```

Comment:
Unlock and lock functions under the hood use approve method with different amount parameter, in case of unlocking we use the max value of the uint256 type, in case of locking we use 0 as amount parameter, I think it is related to this